numpy/lib/recfunctions.py

```diff
@@ -26,10 +26,12 @@
 
 
 __all__ = [
-    'append_fields', 'drop_fields', 'find_duplicates',
-    'get_fieldstructure', 'join_by', 'merge_arrays',
-    'rec_append_fields', 'rec_drop_fields', 'rec_join',
-    'recursive_fill_fields', 'rename_fields', 'stack_arrays',
+    'append_fields', 'apply_along_fields', 'assign_fields_by_name',
+    'drop_fields', 'find_duplicates', 'get_fieldstructure',
+    'join_by', 'merge_arrays', 'rec_append_fields',
+    'rec_drop_fields', 'rec_join', 'recursive_fill_fields',
+    'rename_fields', 'require_fields', 'stack_arrays',
+    'structured_to_unstructured',
```

Comment:
For reviewers: added `apply_along_fields`, `assign_fields_by_name`, and `structured_to_unstructured`

Reply:
and `require_fields`